src/source/image_source.js

```diff
@@ -137,7 +137,7 @@ class ImageSource extends Evented implements Source {
      * Updates the image URL and, optionally, the coordinates. To avoid having the image flash after changing,
      * set the `raster-fade-duration` paint property on the raster layer to 0.
      *
-     * @param {Object} options
+     * @param {Object} options Options object.
```

Comment:
We have the type annotations, should we use that instead of `Object`?

Reply:
I think that's a good long-term goal, but I think it's out of scope for this change. Or is there a technique you know of to link type annotations automatically?